### PR TITLE
dhd: Create /data. Fixes JB#47683

### PR DESCRIPTION
**Pull request by ibaranovomp**

### CODE REVIEW

File: droid-hal-device.inc [hunk 1]
```diff
@@ -656,7 +657,6 @@ _remove_cruft   "/fstab.*"          \
                 "/seapp_contexts"   \
                 "/charger"          \
                 "/res"              \
-                "/data"             \
```
Comment (ibaranovomp) [inline droid-hal-device.inc]: In fact, I'm not sure that we need to remove the /data cleanup before it creation. Theoretically there can be some garbage.

File: droid-hal-device.inc [hunk 2]
```diff
@@ -815,6 +815,7 @@ fi
 
 %files -f tmp/droid-hal.files
 %defattr(-,root,root,-)
+/data
```
Comment (saukko) [inline droid-hal-device.inc]: This should be %dir /data to ensure we only include the directory for this package, right?

File: droid-hal-device.inc (hunk 1 above)
Comment (saukko) [reply droid-hal-device.inc]: I would just remove it just to be on the safe side.

Comment (ibaranovomp) [reply droid-hal-device.inc]: Done

File: droid-hal-device.inc (hunk 2 above)
Comment (ibaranovomp) [reply droid-hal-device.inc]: Done

**Comment by ibaranovomp** [comment]
UP

**Pull request merged**